mkdocs.yml

```diff
@@ -1,7 +1,28 @@
 site_name: LORIS
 repo_url: https://github.com/aces/Loris
-pages:
-    - Home: index.md
-    - API: API/LorisRESTAPI_v0.0.3.md
-    - Wiki: wiki/index.md
+nav:
+    - Home: 'index.md'
+    - Wiki: 'wiki/index.md'
+    - Server Install and Configuration: 
+        - 'Prerequisities and Assumptions': 'wiki/00 - SERVER INSTALL AND CONFIGURATION/00 - Prerequisites And Assumptions/README.md'
+        - Installation:
+            - 'Ubuntu Install': 'wiki/00 - SERVER INSTALL AND CONFIGURATION/01 - LORIS Install/Ubuntu/README.md'
+            - 'CentOS Install': 'wiki/00 - SERVER INSTALL AND CONFIGURATION/01 - LORIS Install/CentOS/README.md'
+            - 'Mac Install (for development)': 'wiki/00 - SERVER INSTALL AND CONFIGURATION/01 - LORIS Install/Macintosh/README.md'
+        - Website Configuration: 'wiki/00 - SERVER INSTALL AND CONFIGURATION/02 - Website Configuration/README.md'
+    - Study Parameters Setup:
+        - Study Variables: 
+            - 'Introduction': 'wiki/01 - STUDY PARAMETERS SETUP/01 - Study Variables/00 - Introduction to Study Variables.md'
+            - 'Projects': 'wiki/01 - STUDY PARAMETERS SETUP/01 - Study Variables/02 - Projects.md'
+            - 'Sites': 'wiki/01 - STUDY PARAMETERS SETUP/01 - Study Variables/03 - Sites.md'
+        - Clinical Instruments: 'wiki/01 - STUDY PARAMETERS SETUP/02 - Clinical Instruments/README.md'
+        - Modules: 'wiki/01 - STUDY PARAMETERS SETUP/03 - Loris Modules/README.md'
+
+            
+    - Getting Started: 'wiki/02 - GETTING STARTED/README.md'
+    - Help and Troubleshooting: 'wiki/03 - HELP AND TROUBLESHOOTING/README.md'
+    - API: 'API/LorisRESTAPI_v0.0.3.md'
```

Comment:
v0.0.3 is still officially an unstable dev version, this should be linking to 0.0.2 (though we really should release 0.0.3..)